CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ include(CTest)
 
 set(CMAKE_CXX_STANDARD 20)
 set(CXX_STANDARD_REQUIRED ON)
+set(CMAKE_POLICY_DEFAULT_CMP0063 NEW)
```

Comment:
Please leave a comment here kn what this means since I know we will forget this immediately, waiting for other pr in any case.